client/web/reporter/src/views/Report/Builder.vue

```diff
@@ -651,21 +656,24 @@ export default {
       handler (reportID) {
         this.unsavedBlocks.clear()
         this.scenarios.selected = undefined
+        this.reportBlocks = []
         this.report = undefined
         if (reportID) {
           this.processing = true
+          this.fetchingReport = true
 
           this.fetchReport(this.reportID)
             .then(() => {
               this.mapBlocks()
             }).catch(() => {
-              this.report = undefined
-              this.reportBlocks = []
               this.toastErrorHandler(this.$t('notification:report.loadFailed'))
-              this.$router.go(-1)
+              this.$router.push({ name: 'report.list' })
             })
             .finally(() => {
               this.processing = false
```

Comment:
You need to include the processing in the timeout since otherwise the topbar and toolbar flash